src/devtools/controller.ts

```diff
@@ -5,6 +5,8 @@ import {
   SetOverridesMessage,
 } from "../../devtools";
 
+const isProd = process.env.NODE_ENV === "production";
```

Comment:
What if we had something like:

```ts
const isChrome = typeof chrome !== 'undefined'
```

My concern with removing the dev build is that production builds take longer to build than dev, so I wouldn't want us to remove the ability to create a dev build.

I'd rather we retain the ability to build both and instead add a script like `serve:devtools` where we serve the devtools. Similarly for popup and options, we'd need a `serve:options` and `serve:popup` or maybe we can have a single webpack-dev-server run serve all 3. Or maybe we serve them all on the same server at the same host but at different paths, e.g. `/devtools.html`, `popup.html`, `options.html`.

Reply:
Regarding this, I see in the screenshot that the dev server hosts all 3, disregard that part.